package.json

```diff
@@ -20,7 +20,7 @@
     "@reduxjs/toolkit": "^1.8.1",
     "@secured-finance/sf-client": "0.2.0-beta.182",
     "@secured-finance/sf-core": "0.2.0-beta.182",
-    "@secured-finance/sf-graph-client": "0.2.0-beta.179",
+    "@secured-finance/sf-graph-client": "^0.2.0-beta.183",
```

Comment:
we removed the ^ here on Aki's suggestion.

Reply:
looks okay apart from the above comment

Reply:
Fixed this